docs/runtime/nodejs-metrics.md

```diff
@@ -0,0 +1,149 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Node.js
+--->
+
+# Semantic Conventions for Node.js Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+This document describes semantic conventions for Node.js metrics in OpenTelemetry.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Experimental](#experimental)
+  - [Metric: `nodejs.active_handles.count`](#metric-nodejsactive_handlescount)
+  - [Metric: `nodejs.active_requests.count`](#metric-nodejsactive_requestscount)
+  - [Metric: `nodejs.eventloop.lag`](#metric-nodejseventlooplag)
+  - [Metric: `nodejs.gc.duration`](#metric-nodejsgcduration)
+  - [Metric: `nodejs.memory.size`](#metric-nodejsmemorysize)
+
+<!-- tocstop -->
+
+## Experimental
+
+**Status**: [Experimental][DocumentStatus]
+
+**Description:** Experimental Node.js metrics captured under `nodejs.`
+
+### Metric: `nodejs.active_handles.count`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.nodejs.active_handles.count(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `nodejs.active_handles.count` | UpDownCounter | `{handles}` | Number of active handles. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+<!-- semconv metric.nodejs.active_handles.count(full) -->
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`nodejs.version`](/docs/attributes-registry/nodejs.md) | string | Node.js version. | `v22.0.0`; `v21.7.3` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+### Metric: `nodejs.active_requests.count`
```

Comment:
What exactly is an `active_request` here?

Reply:
Number of active libuv requests